.cruft.json

```diff
@@ -0,0 +1,20 @@
+{
+  "template": "https://github.com/networktocode-llc/cookiecutter-ntc",
+  "commit": "83ae54f27bbeb3ca93326ab1867052ed9d3f6b6a",
```

Comment:
can't we use the "latest" one?

Reply:
Very good point, I was able to use `checkout: main` instead.